build.py

```diff
@@ -2280,102 +2431,104 @@ def enable_all():
+        )
         fail_if(
             parts[0]
-            not in ['base', 'gpu-base', 'pytorch', 'tensorflow',
-                    'tensorflow2'], 'unsupported value for --image')
+            not in ["base", "gpu-base", "pytorch", "tensorflow", "tensorflow2"],
+            "unsupported value for --image",
+        )
         log('image "{}": "{}"'.format(parts[0], parts[1]))
-        if parts[0] == 'tensorflow2':
-            parts[0] = 'tensorflow'
+        if parts[0] == "tensorflow2":
+            parts[0] = "tensorflow"
         images[parts[0]] = parts[1]
 
     # Initialize map of library paths for each backend.
     library_paths = {}
     for lpath in FLAGS.library_paths:
-        parts = lpath.split(':')
+        parts = lpath.split(":")
         if len(parts) == 2:
             log('backend "{}" library path "{}"'.format(parts[0], parts[1]))
-            if parts[0] == 'tensorflow2':
-                parts[0] = 'tensorflow'
+            if parts[0] == "tensorflow2":
+                parts[0] = "tensorflow"
             library_paths[parts[0]] = parts[1]
 
     # Parse any explicitly specified cmake arguments
     for cf in FLAGS.extra_core_cmake_arg:
-        parts = cf.split('=')
-        fail_if(
-            len(parts) != 2,
-            '--extra-core-cmake-arg must specify <name>=<value>')
+        parts = cf.split("=")
+        fail_if(len(parts) != 2, "--extra-core-cmake-arg must specify <name>=<value>")
         log('CMake core extra "-D{}={}"'.format(parts[0], parts[1]))
         EXTRA_CORE_CMAKE_FLAGS[parts[0]] = parts[1]
 
     for cf in FLAGS.override_core_cmake_arg:
-        parts = cf.split('=')
+        parts = cf.split("=")
         fail_if(
-            len(parts) != 2,
-            '--override-core-cmake-arg must specify <name>=<value>')
+            len(parts) != 2, "--override-core-cmake-arg must specify <name>=<value>"
+        )
         log('CMake core override "-D{}={}"'.format(parts[0], parts[1]))
         OVERRIDE_CORE_CMAKE_FLAGS[parts[0]] = parts[1]
 
     for cf in FLAGS.extra_backend_cmake_arg:
-        parts = cf.split(':', 1)
+        parts = cf.split(":", 1)
         fail_if(
             len(parts) != 2,
-            '--extra-backend-cmake-arg must specify <backend>:<name>=<value>')
+            "--extra-backend-cmake-arg must specify <backend>:<name>=<value>",
+        )
         be = parts[0]
-        parts = parts[1].split('=', 1)
+        parts = parts[1].split("=", 1)
         fail_if(
             len(parts) != 2,
-            '--extra-backend-cmake-arg must specify <backend>:<name>=<value>')
+            "--extra-backend-cmake-arg must specify <backend>:<name>=<value>",
+        )
         fail_if(
             be not in backends,
-            '--extra-backend-cmake-arg specifies backend "{}" which is not included in build'
-            .format(be))
+            '--extra-backend-cmake-arg specifies backend "{}" which is not included in build'.format(
+                be
+            ),
```

Comment:
Yet another separate follow-up idea since the PR is huge, but I think standardizing to f-strings would be nice and remove some extra lines of code throughout. Looks like there's a tool to automate replacement: https://github.com/ikamensh/flynt that may help.

Reply:
Great idea! Created DLIS-5167 to track.